source/write/delete.txt

```diff
@@ -0,0 +1,182 @@
+
+You can modify the behavior of the ``delete_one()`` and ``delete_many()`` methods by
+passing an instance of the ``mongocxx::options::delete_options`` class as an optional
+parameter. The following table describes the fields you can set in a
+``mongocxx::options::delete_options`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Field
+     - Description
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``write_concern``
+     - | Sets the write concern for the operation.
+         For more information, see :manual:`Write Concern </reference/write-concern/>`
+         in the {+mdb-server+} manual.
+
+   * - ``hint``
+     - | Gets or sets the index to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/delete/#std-label-deletes-array-hint>`
+         in the {+mdb-server+} manual.
+
+   * - ``let``
+     - | Specifies a document with a list of values to improve operation readability.
+         Values must be constant or closed expressions that don't reference document
+         fields. For more information, see the :manual:`let statement
+         </reference/command/delete/#std-label-delete-let-syntax>` in the
+         {+mdb-server+} manual.
+
+   * - ``comment``
+     - | Attaches a comment to the operation. For more information, see the :manual:`delete command
+         fields </reference/command/delete/#command-fields>` guide in the
+         {+mdb-server+} manual.
+
+The following code uses the ``delete_many()`` method to delete all documents in
+the ``restaurants`` collection that have a ``name`` value containing the string ``"Mongo"``.
+It also sets the ``comment`` field of a ``mongocxx::options::delete_options`` instance
+to add a comment to the operation:
+
+.. literalinclude:: /includes/write/delete.cpp
+   :start-after: start-delete-options
+   :end-before: end-delete-options
+   :language: cpp
+   :dedent:
+
+.. tip::
+
+   If the preceding example used the ``delete_one()`` method instead of
+   ``delete_many()``, the driver would delete only the first document that has
+   a ``name`` value containing ``"Mongo"``.
+
+Return Value
+~~~~~~~~~~~~
+
+The ``delete_one()`` and ``delete_many()`` methods return an instance of
+the ``mongocxx::result::delete_result`` class. This class contains the
+following member functions:
+
+- ``result()``, which returns the raw bulk write result
+- ``deleted_count()``, which returns the number of documents deleted
+
+If the query filter does not match any documents, the driver doesn't delete any
+documents and ``deleted_count`` is 0.
+
+The following example uses the ``delete_many()`` method to delete documents
+that have a ``cuisine`` value of ``"Greek"`` and prints the number of deleted
+documents:
```

Comment:
S: Split this to sentence and point out that the `deleted_count()` function is being used
```suggestion
The following example calls the ``delete_many()`` method to delete documents
that have a ``cuisine`` value of ``"Greek"``. It then calls the ``deleted_count()`` member function to print the number of deleted
documents:
```